Build failure

https://circleci.com/gh/yolothreat/utilitybelt/38 failed:

```
py.test --cov utilitybelt tests/tests.py
============================= test session starts ==============================
platform linux2 -- Python 2.7.6 -- py-1.4.26 -- pytest-2.7.0
rootdir: /home/ubuntu/utilitybelt, inifile: 
plugins: cov
collected 18 items 

tests/tests.py ..............F...

=================================== FAILURES ===================================
__________________________ TestUB.test_urlvoid_check ___________________________

self = <tests.TestUB testMethod=test_urlvoid_check>

    def test_urlvoid_check(self):
        self.assertIsNone(ub.urlvoid_check('asdf'))
        com_data = ub.urlvoid_check('github.com')
        self.assertIsInstance(com_data, dict)
>       self.assertIn('SCUMWARE', com_data)
E       AssertionError: 'SCUMWARE' not found in {}

tests/tests.py:145: AssertionError
--------------- coverage: platform linux2, python 2.7.6-final-0 ----------------
Name          Stmts   Miss  Cover
---------------------------------
utilitybelt     175      5    97%

===================== 1 failed, 17 passed in 3.90 seconds ====================== py.test --cov utilitybelt tests/tests.py returned exit code 1
```

--- a/utilitybelt.py
+++ b/utilitybelt.py
@@ -300,7 +300,7 @@ def urlvoid_check(name):
     data = BeautifulSoup(response.text)
     if data.findAll('div', attrs={'class': 'bs-callout bs-callout-info'}):
         return None
-    elif data.findAll('div', attrs={'class': 'bs-callout bs-callout-warning'}):
+    elif data.findAll('div', attrs={'class': 'bs-callout bs-callout-danger'}):
         for each in data.findAll('img', alt='Alert'):
             detect_site = each.parent.parent.td.text.lstrip()
             detect_url = each.parent.a['href']

